src/index.html

```diff
@@ -9,6 +9,88 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+
+    <header class="header">
+
+      <a class="header_logo-link" href="#home">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
```

Comment:
I'd say it is not necessary to move those to a new line. Wouldn't say it is a mistake though